### PR TITLE
Pal 148 palisade service testing verification

### PR DESCRIPTION
**Pull request by dev930018**

### CODE REVIEW

File: palisade-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/palisade/mock/AuditServiceMock.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package uk.gov.gchq.palisade.integrationtests.palisade.mock;
```
Comment (developer01189998819991197253) [inline palisade-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/palisade/mock/AuditServiceMock.java]: License and header

File: palisade-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/palisade/mock/PalisadeServiceMock.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
+package uk.gov.gchq.palisade.integrationtests.palisade.mock;
```
Comment (developer01189998819991197253) [inline palisade-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/palisade/mock/PalisadeServiceMock.java]: License and Header

File: palisade-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/palisade/mock/PolicyServiceMock.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
+package uk.gov.gchq.palisade.integrationtests.palisade.mock;
```
Comment (developer01189998819991197253) [inline palisade-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/palisade/mock/PolicyServiceMock.java]: License and Header

File: palisade-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/palisade/mock/ResourceServiceMock.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
+package uk.gov.gchq.palisade.integrationtests.palisade.mock;
```
Comment (developer01189998819991197253) [inline palisade-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/palisade/mock/ResourceServiceMock.java]: License and Header

File: palisade-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/palisade/mock/UserServiceMock.java [hunk 5]
```diff
@@ -0,0 +1,33 @@
+package uk.gov.gchq.palisade.integrationtests.palisade.mock;
```
Comment (developer01189998819991197253) [inline palisade-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/palisade/mock/UserServiceMock.java]: License and Header

File: palisade-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/palisade/mock/StubResource.java [hunk 6]
```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2019 Crown Copyright
```
Comment (ac74475) [inline palisade-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/palisade/mock/StubResource.java]:
```suggestion
 * Copyright 2020 Crown Copyright
```

File: palisade-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/palisade/service/PalisadeComponentTest.java [hunk 7]
```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2019 Crown Copyright
```
Comment (ac74475) [inline palisade-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/palisade/service/PalisadeComponentTest.java]:
```suggestion
 * Copyright 2020 Crown Copyright
```

File: palisade-service-integration/src/test/resources/application.yaml [hunk 8]
```diff
@@ -0,0 +1,68 @@
+# Copyright 2019 Crown Copyright
```
Comment (ac74475) [inline palisade-service-integration/src/test/resources/application.yaml]:
```suggestion
# Copyright 2020 Crown Copyright
```

**Pull request merged**